### ISSUE TITLE
8541 when calling Teams user

### ISSUE THREAD

**Issue by martenekblad** [microsoftgraph/microsoft-graph-comms-samples#343]
We receives reports from some customers that they do not receive calls but the agents are logged on and is ready to receive calls.
Digging into the logs I see that they are reported from the API with:
ResponseCode=400, SubCode=8541, ResponseText=Callee only had Trouter endpoints registered and PNH got delivery errors from all of them.. DiagCode: 480#10076

What could be the reason for this?

Seems to be related to specific users during a specific period of time.

**Comment by TeamMateTech** [comment]
Hi @martenekblad  - were you able to figure out what that message is all about?

**Comment by vrallapalli034** [comment]
Hi @martenekblad - were you able to find out what is causing that issue?  Thanks

**Comment by martenekblad** [comment]
No. Not really. We have seen it in lab when switching tenant. But I'm sure that it's not the case with our customers. Customers are given the recommendation to restart the Teams client and that seems to help.

**Comment by vrallapalli034** [comment]
Thank you @martenekblad for your quick response.

**Comment by LizzieDoo** [comment]
Hello,

Did someone find out about the cause of this issue ;Callee only had Trouter endpoints registered and PNH got delivery errors from all of them." on a teams users?
